wagtail/wagtailadmin/static_src/wagtailadmin/scss/components/_streamfield.scss

```diff
@@ -147,56 +171,59 @@ li.sequence-member {
 // Controls for ordering and deletion of items in a sequence
 .sequence-controls {
     @include transition(opacity 0.2s ease);
-    border-radius: 2px 0 0;
-    border: 1px solid darken($color-input-focus, 10%);
-    border-width: 1px 1px 0;
-    opacity: 0;
-    visibility: hidden;
-    position: absolute;
-    top: -30px;
+    height: 30px;
+    overflow: visible;
     right: 0;
+    width: 100%;
     z-index: 1;
-    overflow: visible;
-    background-color: $color-input-focus;
-    padding: 0 0 0 1em;
-    height: 30px;
 
     h3 {
-        margin: 0;
-        font-size: 0.7rem;
         display: inline-block;
-        margin-right: 1em;
+        font-size: 0.8rem;
         line-height: 1.8rem;
+        padding: 0 20px;
 
         label {
-            color: $color-teal;
-            font-weight: normal;
-            text-transform: uppercase;
             float: none;
-            width: auto;
-            padding: 0;
+            font-weight: bold;
             margin: 0;
+            padding: 0;
+            text-transform: uppercase;
+            width: auto;
         }
     }
 
+    // overrides form.scss
     button.icon.text-replace {
+        background-color: transparent;
+        border: 0;
+        color: $color-teal;
+        padding: 0 20px;
```

Comment:
Could you refactor this so the icon is centered in the button's clickable area? See outline below.

<img width="159" alt="streamfield-controls" src="https://user-images.githubusercontent.com/877585/33045077-36edd3ec-ce54-11e7-8a75-6374f0ef4140.png">

Reply:
👍